contracts/migration/localhost/Aggregator/20230417164547_ADA-USDT.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
We will need to remove the above mentions properties from all localhost migration files 🙏